repos/system_upgrade/common/files/distro/centos/gpg-signatures.json

```diff
@@ -1,8 +1,14 @@
 {
     "keys": [
         "24c6a8a7f4a80eb5",
+        "4eb84e71f2ee9d55",
```

Comment:
I will probably still add these to key-rpms, but I have yet to find the files somewhere.